libraries/c_sdk/standard/mqtt/src/iot_mqtt_operation.c

```diff
@@ -1087,6 +1089,129 @@ void _IotMqtt_ProcessSend( IotTaskPool_t pTaskPool,
 
 /*-----------------------------------------------------------*/
 
+void _IotMqtt_ManagedMqttProcessSend( _mqttOperation_t * pOperation )
+{
+    bool destroyOperation = false, waitable = false, networkPending = false;
+    _mqttConnection_t * pMqttConnection = pOperation->pMqttConnection;
+
+    /* The given operation must have an allocated packet and be waiting for a status. */
+    IotMqtt_Assert( pOperation->u.operation.status == IOT_MQTT_STATUS_PENDING );
```

Comment:
Do you want to assert if pOperation is NULL as well?

Reply:
This has already been checked while creating an operation.